src/layouts/buttonsContainer/styles.css

```diff
@@ -0,0 +1,17 @@
+
+.container{
+    grid-area: 'buttons';
```

Comment:
buttons not 'buttons'. please delete quotation marks.

Currently we have invalid property value